Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_finalny_model.ipynb

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
<p>fajnie, że opisujecie parametry</p><p> </p><p>n_jobs to liczba wątków używanych do trenowania. To nie jest hyperparametr</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>